_docs/collators.md

```diff
@@ -31,9 +31,43 @@ If you have any questions or run into issues, head over to the [OAK Discord](htt
 | Leave candidacy duration `const parachainStaking.leaveCandidatesDelay`                    | 24 rounds or ~48 hours                                     |
 | Reduction of self-delegation bond duration `const parachainStaking.revokeDelegationDelay` | 24 rounds or ~48 hours                                     |
 | Rewards payout `const parachainStaking.rewardPaymentDelay`                                | Time left to complete current round + 2 rounds or ~4 hours |
+| Inflation `parachainStaking.inflationConfig`                                | 5.00% |
```

Comment:
Could point out that it's 5% annual, since there are a couple different inflation percents.